metrics/interface/src/main/assembly/resources/conf/iotdb-metric.yml

```diff
@@ -20,10 +20,11 @@
 # whether enable the module
 enableMetric: false
 
-# can be multiple reporter, e.g., jmx, prometheus
+# can be multiple reporter, e.g., jmx, prometheus, iotdb
 metricReporterList:
   - jmx
   - prometheus
+  - iotdb
```

Comment:
from my perspective I think IoTDBReporterConfig can be config in this properties?